ios/LocalNotifyPlugin.mm

```diff
@@ -30,7 +30,7 @@ - (void) cancelOldNotifications {
 		UILocalNotification *n = [notifications objectAtIndex:ii];
```

Comment:
new changes are described here
https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/SchedulingandHandlingLocalNotifications.html#//apple_ref/doc/uid/TP40008194-CH5-SW1 
i dont know much of ios part but seems alot of things need to change,please look